prow/config.yaml

```diff
@@ -6108,6 +6108,31 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--clean"
         - "--timeout=45"
+  kubernetes-sigs/gcp-compute-persistent-disk-csi-driver:
+  - name: pull-gcp-compute-persistent-disk-csi-driver-test
+    context: pull-gcp-compute-persistent-disk-csi-driver-test
+    always_run: true
+    rerun_command: "/test pull-gcp-compute-persistent-disk-csi-driver-test"
+    trigger: "(?m)^/test( all| pull-gcp-compute-persistent-disk-csi-driver-test),?(\\s+|$)"
+    agent: kubernetes
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--clean"
+        - "--timeout=45"
+        env:
+        - name: PROJECT
+          value: dyzz-test
```

Comment:
This may be an issue later if you start to get concurrent PRs. @krzyzacy another potential usecase for https://github.com/kubernetes/test-infra/issues/8427

Reply:
yep, this is just a stopgap for now. I am planning to use the boskos client library to loan a project when the tests are being run in CI

Reply:
yeah @BenTheElder this is the use case I talked about yesterday :-)